app/assets/javascripts/wizard/addon/templates/step.gjs

```diff
@@ -0,0 +1,52 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import RouteTemplate from "ember-route-template";
+import getUrl from "discourse-common/lib/get-url";
+import WizardCanvas from "../components/wizard-canvas";
+import WizardStep from "../components/wizard-step";
+
+export default RouteTemplate(
+  class extends Component {
+    @service router;
+
+    <template>
+      {{#if this.showCanvas}}
+        <WizardCanvas />
+      {{/if}}
+
+      <WizardStep
+        @step={{@model.step}}
+        @wizard={{@model.wizard}}
+        @goNext={{this.goNext}}
+        @goBack={{this.goBack}}
+      />
+    </template>
```

Comment:
did that nested class structure trip up sort-class-members? (not that it needs much to be tripped up :P)